client.go

```diff
@@ -146,6 +146,9 @@ var nilDialer = *DefaultDialer
 // non-nil *http.Response so that callers can handle redirects, authentication,
 // etcetera. The response body may not contain the entire response and does not
 // need to be closed by the application.
+//
+// Note:Both Dialer.Subprotocols and requestHeader support the filling of subprotocol.
```

Comment:
This needs more detail to be useful documentation. What needs?

"Filling" should be "allow you to define sub-protocols."